src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasServer.kt

```diff
@@ -71,17 +72,24 @@ fun main(args: Array<String>) = mainBody {
     val parser = ArgParser(args)
     val arg = CosmasServerArgs(parser)
     LOG.info("Try to bind in port ${arg.port}")
+    val freeBucket = arg.freeBucket
+    val paidBucket = arg.paidBucket
+    val gsutilImageName = arg.gsutilImageName
     val server =
-            if (arg.bucket != "") {
+            if (freeBucket != null && paidBucket != null) {
                 if (arg.certChain != null && arg.privateKey != null) {
-                    CosmasServer(arg.port, CosmasGoogleCloudService(arg.bucket),
-                            File(arg.certChain), File(arg.privateKey))
+                    CosmasServer(arg.port,

```

Comment:
Log a message saying if we're using secure on insecure mode

Reply:
done